cdap-ui/app/cdap/components/EntityCard/EntityCard.scss

```diff
@@ -189,6 +189,16 @@ $jump-button-container-size: 80px;
       }
     }
   }
+  
```

Comment:
nit: @yukiej I think this newline might give a stylelint error. Can you rebase against latest develop and run a build?

Reply:
Just fixed and rebased - thanks!